package-lock.json

```diff
@@ -1,6699 +1,8 @@
 {
     "name": "pull-request-toolkit-action",
     "version": "1.0.0",
-    "lockfileVersion": 2,
```

Comment:
You should use Node 16 for generating the package lock so that the lockfile version is not lowered.